test/actions/run_audio.ts

```
@@ -163,4 +163,5 @@
     },
     langs: ["fr", "de"],
   });
+  console.log(listLocalizedAudioPaths(context, "de"));
```

ESLint

[error] Unexpected console statement.

(no-console)